\section*{Discussion}

%The discussion should begin with a succinct statement of the principal findings, outline the strengths and weaknesses of the study, discuss the findings in relation to other studies and the wider (clinical) relevance of the findings, provide possible explanations and indicate questions which remain to be answered in future research.

%%%%%%%%%%%%%%%%%%%%%%%%% restatement of your research question, followed by a statement about whether or not, and how much, your findings "answer" the question. 
Miscarriages are frequent events with a complex aetiology whose genetic components have not been completely understood. Here we develop a scalable pipeline that investigates genetic variation scantily considered in the context of miscarriages. Currently, genetic analyses of miscarriages mostly focuses on detecting chromosomal aneuploidies and large chromosomal aberrations (which explain less than half of the cases) leaving unexplored small size genetic variants, the most abundant type of genomic variation. To some extent small genetic variants have been considered in a number of cases that performed target resequencing of candidate genes, a valid but not systematic approach that does not fully exploit genomic information. 

We developed\gp, a pipeline that prioritized 439 putatively causative variants starting form 11M variants discovered by whole-genome sequencing ten euploid  miscarried embryos. Through systematic investigation of all coding regions \gp selected about 47 genes per embryo taking into account the properties of the genetic variants within genes and prior knowledge of genes implicated in miscarriages. By manual curation of the selected genes we highlight a few cases among which three cases of great relevance for embryonic development. \textit{STAG2} is known in literature for its role in cancer, congenital, and developmental disorders through X-linked recessive patterns of inheritance. \textit{TLE4} is found to interact with the genomic region on chromosome 9 that has been associated with prevalence of miscarriages in a genome-wide study on mothers. \textit{TLE4} appears to be a key gene in embryonic development, as it is expressed in both embryonic and extraembrionc tissues where it participates in the Wnt and Notch pathways. Finally, \textit{FMNL2} is involved in cell motility with a major role in driving cell migration.  


%%%%%%%%%%%%%%%%%%%%%Relate your findings to the issues you raised in the introduction. Note similarities, differences, common or different trends.  Show how your study either corraborates, extends, refines, or conflicts with previous findings.
%discuss the findings in relation to other studies and the wider (clinical) relevance of the findings,
Compared to previous similar studies our work focuses on a systematic exploration of the genome that combines previous knowledge (through the use of list of candidate genes) with hypothesis-free prioritization, 


make it robust to discovery of mutations in genes known to be associated but also to the identification of novel genes. 



%%%%%%%%%%%%%%%%%%%%%outline the strengths and weaknesses of the study,
Our pipeline is reproducible and easy to scale to larger number. It includes a control population to filter out genes that can be prioritized by chance and it is suitable for cases where it is not possible to rely on a large enough number of sample to performs association analysis. 


-- model system 


-- weakness, exclude non-coding and intermediate size sv 



%%%%%%%%%%%%%%%%%%%%%%% If you have unexpected findings, try to interpret them in terms of method, interpretation, even a restructured hypothesis; in extreme cases, you may have to rewrite your introduction. Be honest about the limitations of your study.



%%%%%%%%%%%%%%%%%%%%%State the major conclusions from your study and present the theoretical and practical implications of your study.

%%%%%%%%%%%%%%%%%%%%%%%%%%%Discuss the implications of your study for future research and be specific about the next logical steps for future researchers.
-- burden analysis larger sample of embryos 



-- risk assessment


-- larger scale project 


%Here we want to understand the requirements for large scale population-based study of genomic sequences of unrelated miscarriages focused at dechipering the contribution of small-size mutation 

%Develop  parents with an explanation of the developmental abnormality, delineated the recurrence risks, and assisted the management of subsequent pregnancies.



%modelli dominante/recessivo /de novo 
%Heterogeneity mutations look at the gene 



 %%%literature: other sequences 
%Analysis of genetic variants from exome data improves the genetic diagnosis of fetal structural anomalies when standard investigations (karyotype testing and chromosomal array) are uninformative, as shown by studies on hundreds of trios in wide ranges of gestational ages, phenotypes detected by ultrasound, and pregnancy outcomes, including livebirths \cite{petrovski2019whole, quinlan2019molecular, lord2019prenatal}. 



%Future prspectives: Calibration? integration of gene expression? non-coding? positive control? Copy number variants ? 

%Despite the its decreasing costs, whole-genome sequencing is not yet applied to the diagnosis of aneuploidies  ... \\
%Rare variants have large effects, natural selection prevent them to become common 
%We developed a pipeline to select cases of PLs in which the genome of the PoC is euploid and the mother does not present obvious comorbidities. These cases are similar to cases of idiopathic miscarriages that can be used to target the identification of small-size lethal genomic variants through whole genome sequencing.\\

%The identification of small variants requires large sample size. We observe the fraction of samples which... therefore we estimate that the number of samples to collect shuold be  X times the number of samples to be sequenced ...  a sample size of ...  is required to .... Figure \ref{fig:fractions}\\

%We also learned something about miscarriages: report aggregate statistics of qfPCR and arrayCGH when will be available.\\ 

%samples not used for sequencing can be used to study chromosomal rearrangements 

%Limitations: \\
%- array CGH: oinversion not visible.  only deletion and duplication but when complex it is impossible to determine the  order of the fragments. Complex chromosomal rearrangements  and Chromoanagenesis that do not involve copy nuber variants can not be identified.\\ 
%- Is it valid price-wise or better do low-coverage sequencing? 
