\section*{Results}

%Unnecessary overlap between tables, figures and text should be avoided.
%Please use subheadings for different sections.

To understand genetic susceptibility to miscarriage we studied the genomes of forty-six spontaneously miscarried embryos. The embryos' gestational age at pregnancy termination, calculated as the interval between the pregnancy termination date and the last menstruation date, ranges from 7.14 to 19.43 weeks (median is 10.3 weeks). Twenty-one embryos classifies as the product of recurrent miscarriages \cite{eshre2018eshre}. The mothers of the embryos are mostly of European origin (87\%) and their median age at the date of collection was 36.7$\pm$5.9 years, with slightly significant higher age in recurrent cases compared to first ones (Figure \ref{fig:embryostats}, Mannâ€“Whitney p-value=0.02). For the mothers of the embryos medical records report no major comorbidities. Folic acid was taken by 71\% of the mothers with no difference between first and recurrent cases (Figure \ref{fig:embryostats}, Chi-square p-value=0.96). Median body mass index and menarche age are comparable between first and recurrent cases, as well as comparable to a group of control women (Figure \ref{fig:embryostats}). Altogether, from the available medical records, we suppose that the recruited mothers of the embryos were in the range of healthy adult individuals. %The nine women with recurrent clinical miscarriage had an average of 4 previous clinical miscarriages (SD=2.7, range 3-11), average age of 33 years (SD=6; range 23-40), average BMI within the range of normality (24.6; SD=3.9), and ovarian reserve in the XX percentile by the mean age (Anti-Mullerian hormone mean 2.1 pmol/l; SD=2.1).

It is known from literature that about half of the miscarriages in the first trimester are due to large chromosomal aneuploidies, such as trisomies or deletions of large chromosomal chunks \cite{pmid22796359}. In this study we want to focus on cases in which the genome is euploid, therefore the forty-six embryos were screened for chromosomal aneuploidies prior to whole-genome sequencing. We found that 32.6\% of samples were euploid and could be sequenced while 56.6\% of the embryos presented aneuploidies (Figure \ref{fig:presequencing}).The most common aneuploidy in our data set is the trisomy of chromosome 22 (26.9\%), followed by trisomy of chromosome 16 (15.4\%). In particular, a first round of detection of aneuploidies on chromosomes 13, 15, 16, 18, 21, 22, X, and Y through Short Tandem Repeats analysis discarded 45.7\% of samples, %These types of repeats (tetra- or penta-nucleotide) are often expected to be found in heterozygosis, therefore triploidy is assumed when three alleles are found at several markers along a chromosome (complete) or part of it (partial). Similarly, uniparental disomy for a targeted region or chromosome is assumed when only one parental allele is amplified. 
and a subsequent analysis through comparative genomic hybridization and copy number variation detection form low-coverage sequencing discarded another 10.9\% of the samples. Finally, a number of embryos (10.9\%) dropped off the analysis due to low-quality DNA or maternal contamination. 

After ascertaining euploidy, %the exome of the nine women was sequenced using Agilent SureSelect whole-exome capture and Illumina sequencing technology on the NovaSeq 6000 Series Sequencer(Next Generation Solutions, Hong Kong, China), while 
the whole genome of ten embryos was sequenced using Illumina short-reads at 30X coverage. In the set of embryos genomes, we identified 11M single-nucleotide polymorphisms (SNPs) and 2M small insertions or deletions (indels).%, while from the exome data of the women we identified 1.7M SNPs and 276k indels.
%In the set of embryos genomes, we identified 11041,557 single-nucleotide polymorphisms (SNPs), 1980256 small insertions or deletions (indels), and XX copy number variants (CNVs). In the set of women we identified 1738895 SNPs and 275652 indels.


%5493 genes in hgdp for exomes
%1323 discarded and 76% retained
\subsection*{Prioritization of genetic variants in coding genomic regions} 
We developed the \gp pipeline to prioritize putatively damaging genetic variants from sequencing data. \gp takes as input genomic variants information from cases and controls (including the \emph{per}-individual allelic counts) in form of a VCF file and outputs a table of variants prioritized according to user-defined parameters. \gp uses functional annotations of genomic variants, information from publicly available sequence data of presumably healthy individuals, and, if available, knowledge of genes involved in the trait under study. \gp currently analyzes coding regions and performs four filtering steps (Figure \ref{fig:pipeline}A). The first filter (Filter I, Figure \ref{fig:pipeline}B) retains variants based on: (i)  an overall impact on the gene product classified as moderate or high\cite{mclaren2016ensembl}: (ii) a user-defined threshold of allele frequency in control populations; (iii) the combined property of being putatively damaging (quantified by the CADD score\cite{rentzsch2019cadd}) and located in genes intolerant to loss of function (determined by the pLI score\cite{karczewski2020mutational}). In addition it is possible to incorporate one or more user-defined lists of genes relevant to the trait under study. Variants retained by Filter I (hits) are further filtered to control for false positives with Filters II and III. In particular Filter II removes variants in genes with too many hits, while Filter III determines the chance for genes to be selected in a control population based on criteria specified in Filter I. In practice, a number of control individuals are sampled a number of times and their genetic data filtered using Filter I to obtain a list of genes selected by chance (Figure \ref{fig:pipeline}C). Finally, Filter IV excludes private variants with read depth outside the range found in non-private ones.   

%Filter II removes variants in genes with too many hits. Filter III controls for the chance of random occurrence of genes based on replicates of Filter I analysis in a large control population. In practice it removes all the genes that pass Filter I a user-defined-fraction of times across a user-defined-number of replicates. 

We applied the \gp pipeline to data from the high-coverage whole-genome sequences of genomic DNA of the embryos. For Filter I we set allele frequency $<$0.05\% in the 1000 Genomes\cite{1000genome2015global} and gnomAD\cite{karczewski2020mutational} reference populations, while the functional effect of the variant within the gene context was taken into account in two ways: either selecting for putatively deleterious variants (CADD score $>$90th percentile) in genes highly intolerant to loss of function (pLI score $>$0.9), or selecting for variants in genes known to be involved in early embryonic development. In particular for this last option we included five lists of genes, namely genes involved in embryo development (Gene Ontology GO:0009790), genes lethal during embryonic stages \cite{dawes2019gene}, essential for embryo development \cite{dawes2019gene}, genes discovered through the Deciphering Developmental Disorders project\cite{study2015large}, and a manually curated list of candidate genes known to be involved in miscarriages. We requested the variant to satisfy one or both these criteria: (i) be in a gene present in at least two of the five lists or (ii) have CADD score above the 90th percentile and be in a gene with pLI$>$0.9. Overall, filter I retained 1,038 variants (hits) in embryos.   

Filter II removed variants in genes with $>$5 hits, under the assumption that variants found in these genes are likely to be sequencing and alignment artifacts. With few exceptions, we observed that the number of hits per gene at the 99th percentile was five, even if there is no significant correlation between number of hits and gene length (Spearman r$^2$=0.05 p-value=0.124), and  that hits in genes with $>$5 hits are enriched for private variants (Figure \ref{fig:filters}A).  

For Filter III we used as control population 929 individuals from the Human Genome Diversity Project\cite{bergstrom2020insights} from which we resampled 100 times ten individuals after checking for population stratification (Figure \ref{fig:pca}). On each resampled set we performed Filter I analysis and recorded the genes that were retained. Overall 5,488 unique genes were retained in controls with different frequencies in samples across replicates (Figure \ref{fig:filters}B ). When considering the 95th percentile, 1,531 genes are found $>$5\% of times across replicates, therefore hits within these genes were removed by Filter III. %Because the criteria used for Filter I were identical between embryos and mothers, and because the numerosity was very similar, we performed resampling only once and applied the outcome to both analyses.

Filter II and III retained 447 hits of which 21\% are private with respect to 1000 Genomes and gnomAD data sets. Despite comparable read depth between private and non-private variants (Figure \ref{fig:filters}C, KS test p-value=0.99, F-test p-value=0.06), to control for possible artifacts due to scanty coverage, we applied a further filter that removes hits that are private and with read depth outside the range found in non-private ones.  
%The genes prioritized in women are slightly enriched for genes belonging to mitochondrial translation initiation (R-HSA-5368286), elongation (R-HSA-5389840), and termination (R-HSA-5419276) pathways (p-value=0.0002, Benjamini-Hochberg adjusted p-value=0.0574, FDR=0.056). This enrichment is due to eight genes selected by \gp in six women due to missense mutations (Table \ref{tab:mito}).  --More about the eight genes %(\textit{MRPL15}, \textit{MRPL34}, \textit{MRPL37}, \textit{MRPS15}, \textit{MRPS21}, \textit{MRPS23}, \textit{MRPS28}, and \textit{OXA1L}) 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Properties and biological significance of the prioritized variants and genes} 

After all filters, \gp prioritizes 439 unique variants in 399 genes that code for 980 transcripts (Supplementary Table 2). % belonging to several protein classes (Figure \ref{fig:protClass}). 
Almost all the prioritized genes (n=378) have an OMIM accession number and 18.8\% of them were not in the lists of candidate genes used by \gp as input during the prioritization, demonstrating that \gp is robust to detection of genes never investigated before in relationship to the phenotype under study. 

Nine genes are involved in the pathway of mitochondrial translation (Reactome identifier R-HSA-5368287) and this number represents a significant 4.9 fold enrichment over random expectations (Supplementary Table 3, p-value=1.45E-04, FDR=0.03). Similarly, we observe overrepresentation of genes involved in cell cycle checkpoints (R-HSA-69620) and signaling by Rho GTPases (R-HSA-194315). With reference to the cellular compartments where the gene product are expressed, we observe a 7.7 fold significant enrichment (p-value 7.82E-04, FDR=0.04) of protein expressed in the mitotic spindle pole or in associated complexes (Supplementary Table 4), among which the product of \textit{STAG2} for which we observe an high-impact mutation in one embryo from this study. Finally, seven genes (\textit{BHLHE40},\textit{DBN1}, \textit{FOXA1}, \textit{HSPD1}, \textit{PLXNA3}, \textit{SLC35A2}, \textit{SRF}) were previously identified as essential genes in copy-number variable regions from the analysis of hundreds of miscarried fetuses \cite{chen2017characterization} % Among them, \textit{HSPD1} for which two embryos from this study share an heterozygous missense mutation
% BHLHE40
% DBN1
% FOXA1
% HSPD1 in two samples 
% PLXNA3
% SLC35A2
% SRF

In the embryos, 4.1\% of the prioritized variants are stop gains/loss, frameshift indels, and variants that disrupt splicing sites, all classified as having high impact on the gene products, while missense mutations prevail among the variant with moderate effect (Figure \ref{fig:resembryo}A, Table S1). Averages per embryos are 48.9$\pm$8.0 genomic variants in 47.8$\pm$7.7 genes coding for 113.5$\pm$24.6 transcripts (Figure \ref{fig:resembryo}B). In almost all prioritized genes, \gp retains only one variant per embryo, with few exception (five cases with two and one with three variants per gene), as shown in Figure \ref{fig:resembryo}B, where the allele dosage and impact are also shown. %--Shared genes/variants 

\subsubsection*{Mutations in \textit{STAG2}, \textit{FLAD1}, \textit{TLE4}, \textit{FRMPD3}, and \textit{FMNL2} in the embryos} 
%validation merigen? 
The male embryo FE130 carries two high-impact mutations in single copy. The first is a one extremely rare T$>$G transversion (rs913664484, G frequency is 4.7e-05 in 42.7k individuals from gnomeAD) at the 5' end of the first intron of the Stromal antigen 2 (\textit{STAG2}) gene. The mutation disrupts a splicing site, therefore having an high impact on the gene product. \textit{STAG2} is located on the X chromosome and its inactivation is the cause of severe congenital and developmental defects in embryos and infants\cite{mullegama2017novo, mullegama2019mutations, aoi2020nonsense, study2015large} as well as chromosomal aneuploidies in several types of human cancers\cite{solomon2011mutational}. Interestingly, only mildly-deleterious mutations have been found in alive human males, while females can carry highly deleterious mutations in heterozygosis\cite{mullegama2019mutations}. \textit{STAG2} codes for the cohesin subunit SA-2 \cite{cuadrado2020specialized}. Cohesins are ring-shaped protein complexes that bring into close proximity two different DNA molecules or two distant parts of the same DNA molecule and are responsible for the cohesion of sister chromatids \cite{mcnicoll2013cohesin}. In mouse, inactivation \textit{Stag2} causes early embryo lethality \cite{de2020essential}. 

The second high-impact mutation of FE130 is a stop gain in the Flavin Adenine Dinucleotide Synthetase 1 (\textit{FLAD1}) gene that is expressed in the mitochondrial DNA where it catalyzes the adenylation of flavin mononucleotide (FMN) to form flavin adenine dinucleotide (FAD) coenzyme\cite{brizio2006over}. The FAD synthase is an essential protein as the products of its activity, the flavocoenzymes play a vital role in many metabolic processes and in fact FAD synthase deficiencies (OMIM #255100) associated with homozygous severe mutations cause death in the first months of life\cite{balasubramaniam2019disorders}. In FE130 the stop mutation p.Q159* affects one of the five isoforms (Uniprot identifier Q8NFF5-5) at the second last residue, therefore we can speculate that it might not seriously compromise the function of the protein.   

The embryo FE136 carries an heterozygous missense mutation (rs41307447) in the Transducin-like enhancer protein 4 gene (\textit{TLE4}, synonym \textit{GRG-4}) that causes a substitution of a polar amino acid with another polar amino acid (Ser$>$Tre) in the seventh exon of the gene, corresponding to a low complexity domain of the protein. The rs41307447 polymorphism is tolerated (SIFT score 0.18) and supposed to be benign (PolyPhen score 0.003), nevertheless the \textit{TLE4} gene is classified as highly intolerant to loss of function (pLI score 0.999) and the CADD score associated to rs41307447 is in the 99.8th percentile. 
\textit{TLE4} is a trascriptional repressor of the Groucho-family expressed in the embryonic stem cells where it represses naive pluripotency gene \cite{laing2015gro} and it is a direct transcriptional target of Notch \cite{menchero2019transitions}. \textit{TLE4} is also expressed in the extravillous trophoblasts \cite{meinhardt2014wnt} where it is part of the Wnt signaling pathway that promotes implantation, trophoblast invasion, and endometrial function \cite{sonderegger2010wnt}. Finally, a study in a cohort of 750 women finds significant association between the A allele of rs7859844 on chromosome 9 and recurrent miscarriages, further showing that rs7859844 physically interact with \textit{TLE4} \cite{laisk2020genetic}. In our study among all embryos only FE106 carries the intergenic variant rs7859844. 

Among prioritized variants shared by more than one embryo, the male FE165 and female FE106 embryos share a stop gain mutation (p.Q1758*) in the X-linked FERM and PDZ domain containing 3 (\textit{FRMPD3}) gene, which is highly intolerant to loss of function (pLI = 0.91). The mutation falls at the protein C-terminal in a polyQ stretch (27 residues). While little is known in humans about this gene, a study in lion head goose finds significant association between high expression of \textit{FRMPD3} and low production of eggs \cite{zhao2020genome}. 

% AS093, AS090, AS087, AS065, AS036) 
Five embryos, among which the carrier of the missense mutation in \textit{TLE4}, share one copy of an haplotype composed of two T alleles 4bp apart causing stop-gain (rs750755379) and missense (rs866373641) substitutions in the Formin-like protein 2 gene (\textit{FMNL2}, Figure \ref{fig:resembryo}C). The two alleles exist at moderate-to-high frequency in human populations (Figure \ref{fig:fmnl2}A) and are in perfect linkage disequilibrium (r\textsuperscript{2}=1) in the embryos. In addition to the two mutations described above, the embryo FE165 has a deleterious and probably damaging missense mutation in phase with the two others (rs189416564, SIFT=0, PolyPhen =0.969). \textit{FMNL2} codes for a formin-related protein expressed in multiple human tissues and in particular in gastrointestinal and mammary epithelia, lymphatic tissues, placenta, and in the reproductive tract\cite{gardberg2010characterization}. In the fetus \textit{FMNL2} is expressed in the cytoplasm of brain, spinal cord, and rectum\cite{lizio2015gateways}. \textit{FMNL2} is an elongation factor of actin filaments that drives cell migration by increasing the efficiency of lamellipodia protrusion \cite{block2012fmnl2, kuhn2015structure}, and its overexpression is associated with cancer \cite{zhu2011fmnl2}. The stop-gain mutation we find in the five embryos is located in the first domain of the protein, a Rho GTPase-binding/formin homology 3 (GBD/FH3) domain involved in subcellular localization and regulation of activation (Figure \ref{fig:fmnl2}B). The stop codon produces a truncated protein that lacks the Formin Homology-2 (FH2) domain, which directly binds to the actin filament catalyzing its nucleation and elongation.